libraries/percentInView/percentInView.js

```diff
@@ -1,6 +1,26 @@
+import { startAuction } from '../../src/prebid.js';
 
-function getBoundingBox(element, {w, h} = {}) {
-  let {width, height, left, top, right, bottom} = element.getBoundingClientRect();
+const cache = new Map();
+
+startAuction.before((next, auctionConfig) => {
+  clearCache();
+  next(auctionConfig);
+});
+
+export function clearCache() {
+  cache.clear();
+}
+
+export function getBoundingBox(element, {w, h} = {}) {
```

Comment:
most of the consumers of this are only interested in the bounding rect and do not need the `w/h` adjustment or the rest of this library. IMO it makes sense to extract it into a different library.

Reply:
why a different library instead of just a different function to import?

Reply:
because importing a function from a library imports the whole library.